README.md

```diff
@@ -14,13 +14,16 @@
 
 LSP4IJ is a free and open-source [Language Server protocol (LSP)](https://microsoft.github.io/language-server-protocol/) client compatible with all flavours of IntelliJ.  
 
-It currently doesn't provide any useful functionality on its own, but is instead used as a dependency for other extensions, willing to integrate language servers with IntelliJ products.
+It allows you to integrate any `language server`:
 
-LSP4IJ provides:
-
-* Language server `extension points` to add any language server:
+* by `developing an IntelliJ plugin` with LSP `extension points`:
   * [com.redhat.devtools.lsp4ij.server](./docs/DeveloperGuide.md#declare-server-with-extension-point) extension point to define a language server.
   * [com.redhat.devtools.lsp4ij.languageMapping](./docs/DeveloperGuide.md#declare-language-mapping-with-extension-point) to associate an IntelliJ language with a language server definition.
+* with [User-defined language server](./docs/UserDefinedLanguageServer.md) which provides the capability to `define a 
```

Comment:
by manually adding [language server definitions](./docs/UserDefinedLanguageServer.md),  supporting custom server settings. This approach doesn't require developing a specific IntelliJ plugin.